Lib/unittest/case.py

```diff
@@ -338,6 +339,25 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def short_description_from_docstring(text):
+    """
+    Return the test case short description from a docstring.
+
+    Ths docstring is parsed by the standard `pydoc.splitdoc` function
```

Comment:
How about:

```python
def short_description_from_docstring(text):
    # Return the summary line from a docstring.
    # If there is no summary line, return None.
    ...
```

The use of the word "summary line" comes straight from PEP-257.